src/lib/addons/new-relic-definition.ts

```diff
@@ -21,7 +21,7 @@ const newRelicDefinition: IAddonDefinition = {
     name: 'new-relic',
     displayName: 'New Relic',
     description: 'Allows Unleash to post updates to New Relic Event API.',
-    documentationUrl: 'https://docs.getunleash.io/docs/addons/new-relic',
+    documentationUrl: 'https://docs.getunleash.io/reference/integrations',
```

Comment:
New relic does not appear in the list of integrations in our docs

Reply:
Added it to the list of community integrations for now